hooks/useWallet.ts

```diff
@@ -52,20 +55,198 @@ export function useWallet() {
+
+        // ⚠️ NUEVO: Verificar campo origins (crítico)
+        console.log('=== Origins Check ===')
+        if (decoded.payload.origins) {
+          console.log('✅ Origins present:', decoded.payload.origins)
+        } else {
+          console.warn('⚠️ WARNING: Origins field is missing!')
+        }
+
+        if (decoded.payload.payload?.eventTicketObjects) {
+          const ticketObj = decoded.payload.payload.eventTicketObjects[0]
+          console.log('Ticket Object ID:', ticketObj?.id)
+          console.log('Ticket Class ID:', ticketObj?.classId)
+          console.log('Ticket State:', ticketObj?.state)
+          console.log('Has Barcode:', !!ticketObj?.barcode)
+          console.log('Barcode Value:', ticketObj?.barcode?.value)
+
+          // Verify ValidTimeInterval (critical fix - formato ISO 8601)
+          if (ticketObj?.validTimeInterval) {
+            console.log('=== ValidTimeInterval (ISO 8601 Format) ===')
+            const startDate = ticketObj.validTimeInterval.start?.date
+            const endDate = ticketObj.validTimeInterval.end?.date
+
+            console.log('Start date:', startDate)
+            console.log('End date:', endDate)
+
+            // Verificar formato ISO 8601 completo
+            const iso8601Regex =
+              /^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}[+-]\d{2}:\d{2}$/
+            if (startDate) {
+              const isStartValid = iso8601Regex.test(startDate)
+              console.log(
+                'Start format (ISO 8601):',
+                isStartValid ? '✅' : '❌',
+                isStartValid
+                  ? 'Valid'
+                  : 'Invalid - should be YYYY-MM-DDTHH:mm:ss±HH:mm'
+              )
+            }
+            if (endDate) {
+              const isEndValid = iso8601Regex.test(endDate)
+              console.log(
+                'End format (ISO 8601):',
+                isEndValid ? '✅' : '❌',
+                isEndValid
+                  ? 'Valid'
+                  : 'Invalid - should be YYYY-MM-DDTHH:mm:ss±HH:mm'
+              )
+            }
+
+            if (!ticketObj.validTimeInterval.end) {
+              console.warn('⚠️ WARNING: ValidTimeInterval.end is missing!')
+            } else {
+              console.log('✅ ValidTimeInterval.end is present')
+            }
+
+            // Verificar que NO tenga campo "time" separado (incorrecto)
+            if (
+              ticketObj.validTimeInterval.start?.time ||
+              ticketObj.validTimeInterval.end?.time
+            ) {
+              console.warn(
+                '⚠️ WARNING: ValidTimeInterval has separate "time" field (incorrect format)!'
+              )
+              console.warn(
+                'Should use ISO 8601 format: YYYY-MM-DDTHH:mm:ss±HH:mm'
+              )
+            }
+          } else {
+            console.warn('⚠️ WARNING: ValidTimeInterval is missing!')
+          }
+        }
+      } else {
+        console.warn('Could not decode JWT token')
+      }
```

Comment:
Please remove this, FE must never decode a token. Instead in the BE you should add tests to validate the generated token is correct